backend/src/BackendOnlyStdLib/LibCrypto.fs

```diff
@@ -34,7 +34,7 @@ let fns : List<BuiltInFn> =
         (function
         | _, [ DBytes data ] -> MD5.HashData(ReadOnlySpan data) |> DBytes |> Ply
         | _ -> incorrectArgs ())
-      sqlSpec = NotYetImplementedTODO
+      sqlSpec = NotYetImplemented
```

Comment:
Postgres seems to support an MD5 fn. https://www.postgresqltutorial.com/postgresql-string-functions/postgresql-md5/
Is this worth pursuing?

The md5 fn seems built-in; there are fns additionally available in the `pgcrypto` extension that could address some more of our crypto fns (I see at least hmacsha256 and hmacsha1 in there)

generally: if there are fns that _could_ be SqlCompile-able, but aren't functions we'd expect someone to reach for often in a `DB::query`, do you have any feel for what to do? I'm sort of doubtful somewhat will try to `Crypto::md5` _often_ in a query, but could see it.

Reply:
Sorry, I somehow missed that you just removed the extra/unneeded "TODO" word here.

Reply:
> Is this worth pursuing?

Not sure if we want to stay on PG long term, and these functions might damage our ability to move to another platform. So my plan is to not add any more unless there is great user demand.